README.md

```diff
@@ -201,6 +195,23 @@ PayPal authentication occurs in a popup window. However, this causes issues with
 
 PopupBridge solves this problem by allowing [`braintree-web`](https://github.com/braintree/braintree-web) to open the PayPal popup from a secure mini-browser.
 
+Using PayPal in a WebView
+-------------------------
+
+WebView-based checkout flows can accept PayPal with PopupBridge the Braintree JS SDK. For the authentication flow, PayPal requires a popup window—which can be simulated with PopupBridge.
+
+### Setup
+1. Create a web-based checkout that accepts PayPal using Braintree JS v3.9.0 or higher
+2. Show a loading indicator
```

Comment:
Might want to mention _when_ the loading indicator should be shown and hidden, e.g.

```
Show a loading indicator when the PayPal button is clicked
   - Hide the loading indicator in the PayPal tokenize completion callback
```